src/components/ActionModal.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+	<div class="overlay-action-modal" @click.self>
+		<div class="container-action-modal" @click.stop>
+			<h3>{{ title }}</h3>
+			<div v-if="isRename" class="rename">
+				<input id="rename-input" type="text" autocomplete="off">
```

Comment:
Please do not use `id`s on components! This breaks encapsulation, it won't be possible to reuse this component